### PR TITLE
[casr-dojo] Support tool for exporting CASR crash reports to DefectDojo

### PR DESCRIPTION
**Pull request by SweetVishnya**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
         install: |
           export CARGO_TERM_COLOR=always
           export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
-          apt-get update && apt-get install -y gdb pip curl python3.10-dev clang llvm build-essential
+          apt-get update && apt-get install -y gdb pip curl python3.10-dev clang llvm build-essential libssl-dev
```
Comment (SweetVishnya) [inline .github/workflows/main.yml:51]: I should fix build for AARCH64 and RISC-V

**Review by SweetVishnya — COMMENTED** [review]
I should write documentation

File: .github/workflows/main.yml (hunk 1 above)
Comment (SweetVishnya) [reply .github/workflows/main.yml]: Or disable build for RISC-V

File: casr/src/bin/casr-dojo.rs [hunk 2]
```diff
@@ -0,0 +1,853 @@
...
+}
+
+/// Return hash for CASR report.
+///
+/// # Arguments
+///
+/// * `report` - CASR report.
+/// * `name` - CASR report name.
+fn compute_report_hash(report: &CrashReport, name: &str) -> Result<u64> {
+    let stacktrace = report.filtered_stacktrace();
```
Comment (SweetVishnya) [inline casr/src/bin/casr-dojo.rs:427]: bail when stack trace is empty

File: casr/src/bin/casr-dojo.rs [hunk 3]
```diff
@@ -0,0 +1,859 @@
...
+        .into_iter()
+        .map(|(e, report)| {
+            let hash = compute_report_hash(&report, e.to_str().unwrap())?;
+            Ok((e, report, hash))
+        })
+        .collect::<Result<Vec<_>>>()?;
+    let new_casr_reports: Vec<(PathBuf, CrashReport, Option<CrashReport>)> = new_casr_reports
+        .into_iter()
+        .filter(|(_, _, hash)| casr_hash.insert(*hash))
+        .map(|(e, report, _)| {
```
Comment (anfedotoff) [inline casr/src/bin/casr-dojo.rs:817]: Could we unite these two map?

File: casr/src/bin/casr-dojo.rs [hunk 4]
```diff
@@ -0,0 +1,859 @@
...
+    info!(
+        "{} new reports are duplicate",
+        total_reports_cnt - new_casr_reports.len()
+    );
+
+    info!(
+        "Uploading {} new unique CASR reports to DefectDojo",
+        new_casr_reports.len()
+    );
+    let gdb_no_crash = new_casr_reports.iter().any(|(_, _, gdb)| gdb.is_some());
```
Comment (anfedotoff) [inline casr/src/bin/casr-dojo.rs:842]: maybe find  some better name for the bool value and parameter for functions?

**Review by Avgor46 — APPROVED** [review]

**Review by anfedotoff — APPROVED** [review]

**Pull request merged**